projects/ui-components/lf-repository-browser/ILfTreeNodeService.ts

```diff
@@ -37,4 +37,10 @@ export interface LfTreeNodeService {
    * returns - Promise with the parent of the parameter LfTreeNode or undefined if its the root LfTreeNode and does not have a parent
    */
   getParentTreeNodeAsync(treeNode: LfTreeNode): Promise<LfTreeNode | undefined>;
+
+  /**
+   * Gets the LfTreeNode referenced by the identifier. This can be specific to the implementation of the service (i.e. id, path, etc.)
+   * @param id Identifier for the LfTreeNode.
+   */
+  getTreeNodeByIdAsync?(id: string): Promise<LfTreeNode | undefined>;
```

Comment:
is identifier the id of the entry? or it can also be the path, depending on the service?

Reply:
yes, I tried to make that clear in the tsdoc, let me know how I can clarify that

Reply:
It makes sense, maybe rename the parameter as identifier?